src/pybind/mgr/dashboard/frontend/e2e/cluster/monitors.e2e-spec.ts

```diff
@@ -20,4 +21,95 @@ describe('Monitors page', () => {
       expect(monitors.getBreadcrumbText()).toEqual('Monitors');
     });
   });
+  describe('fields check', () => {
+    beforeAll(() => {
+      monitors.navigateTo();
+    });
+
+    it('should check status table is present', () => {
+      // check for table header 'Status'
+      expect(
+        monitors
+          .getLegends()
+          .get(0)
+          .getText()
+      ).toMatch('Status');
+
+      // check for fields in table
+      expect(monitors.getStatusTable().getText()).toMatch('Cluster ID');
+      expect(monitors.getStatusTable().getText()).toMatch('monmap modified');
+      expect(monitors.getStatusTable().getText()).toMatch('monmap epoch');
+      expect(monitors.getStatusTable().getText()).toMatch('quorum con');
+      expect(monitors.getStatusTable().getText()).toMatch('quorum mon');
+      expect(monitors.getStatusTable().getText()).toMatch('required con');
+      expect(monitors.getStatusTable().getText()).toMatch('required mon');
+    });
+
+    it('should check In Quorum and Not In Quorum tables are present', () => {
+      // check for there to be two tables
+      expect($$('cd-table').count()).toEqual(2);
```

Comment:
`$$('cd-table')` should be extracted to a method on `page-helper.po.ts`. [info](http://www.protractortest.org/#/style-guide#page-objects)